src/bindgen/index.md

```diff
@@ -12,26 +12,30 @@ them (so the compiler knows what symbols to use). Enter [bindgen][bindgen].
 
 Bindgen does all the hard work of parsing C-style header files and generating 
 the equivalent Rust function definitions. They've also got a 
-[great tutorial][tut] for getting started. We'll try to focus on the more high 
-level stuff which comes with using a C library from Rust, seeing as the zero 
-cost abstractions and high level way of thingking is probably why you're trying 
-use Rust in a C/C++ codebase in the first place!
+[great tutorial][tut] for getting started. This section focuses mostly on the
+high level stuff required when using a C library from Rust, seeing as the zero
+cost abstractions and high level way of thinking is probably why you're trying
+to use Rust in a C/C++ codebase in the first place!
 
 
 ## Getting Set Up
 
-We'll be building on top of the [great tutorial][tut] linked earlier to write
+> **Note:** `bzip2` must be installed on your system for this section.  
+> **For Windows**, The process for getting bzip2 header files
+> recognized is **unknown**. If you can help, please file an [issue][issue].
+
+You'll build off the [great tutorial][tut] linked earlier to write
```

Comment:
Hmm, I just had a look at the [getting started page] of *The Book* and it looks like they use "we" all over the place. I like the language they use because it feels like the user and the author are going through the book together, which makes you feel more involved and is more approachable.

On the other hand something like "an idiomatic Rust wrapper will be written..." might be a bit too formal for a tutorial. I dunno, what do you think?

[getting started page]: https://doc.rust-lang.org/book/first-edition/getting-started.html